src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml

```diff
@@ -339,6 +351,62 @@
             </section>
         }
 
+        @if (Model.IsAnalyzed && Model.ComputerVisionAnalyzeResponse?.BatchReadResult != null)
+        {
+            <section class="cwb-result-computervision-analyze">
+                <h2 class="mt-5">Computer Vision - Batch Read</h2>
+
+                <div class="row">
+                    <div class="col-md-7">
+                        <div class="card mb-3 cwb-result-image">
+                            <img src="@Model.ComputerVisionAnalyzeResponse.ImageInfo.Url" class="card-img-top" alt="@Model.ComputerVisionAnalyzeResponse.ImageInfo.Description" />
+                            <div class="cwb-image-boxes">
+                                <div class="cwb-image-boxes-ocr-lines">
```

Comment:
Use something unique for this, like `cwb-image-boxes-batchread-lines`

Reply:
Shame on me, I missed this one! I thought I changed them after your comment on PR number 1

Reply:
And I also rollbacked by error the fix on recognizeText class. Everything should be ok right now